unit_tests/test_cluster.py

```diff
@@ -18,6 +18,10 @@
 class DummyNode(BaseNode):  # pylint: disable=abstract-method
     _database_log = None
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.remoter.stop()
+
```

Comment:
I'm completely unfamiliar with this code, so I don't know what I'm reviewing, but how is this change related to a DNS server?

Reply:
this is not related, and I'll remove it, since it seem to not even solve the problem I was trying to fix.